File: ex06/ex06_06/ex06_06.py
from functools import reduce

def odd(x):
    return x % 2 == 1

def sum(a, b):
    return a + b

def main():
    x = list(range(1, 40))
    x = list(filter(odd, x))
    sum_series = reduce(sum, x, 0)
    print(sum_series)

if __name__ == "__main__":
    main()

File: ex06/ex06_06/ex06_06.js
function sequence(x, start, end) {
  for (let i = start; i <= end; i++) {
    x.push(i);
  }
}
function odd(x) {
  return x % 2 === 1;
}
function sum(a, b) {
  return a + b;
}
function main() {
  let x = [];
  sequence(x, 1, 39);
  x = x.filter(odd);
  let sumSeries = x.reduce(sum, 0);
  console.log(sumSeries);
}
main();